.github/workflows/release.yml

```diff
@@ -2,7 +2,7 @@ name: Release
 on:
   push:
     branches:
-      - master
+      - main
```

Comment:
@cs-raj We're using the master in the tsgen plugin; could you please check if it's working or not?

Reply:
Yes @aman19K Fixed it